Assets/Scripts/Core/MeshUtils.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System.Collections;

```

Comment:
This entire file seems duplicated, most of the functions can be found in `Utils.cs`, and any missing functions should go there too.